assets/js/modules/analytics-4/datastore/constants.js

```diff
@@ -32,3 +32,31 @@ export const ENHANCED_MEASUREMENT_FORM = 'enhanced-measurement-form';
 export const ENHANCED_MEASUREMENT_ENABLED = 'enhanced-measurement-enabled';
 export const ENHANCED_MEASUREMENT_SHOULD_DISMISS_ACTIVATION_BANNER =
 	'enhanced-measurement-should-dismiss-activation-banner';
+
+export const customDimensions = {
```

Comment:
If I'm not wrong, it will be used in other places as well. @nfmohit, do you have any reference that it will be used in other places, or should we move it to `custom-dimensions.js`?

Reply:
Hi @hussain-t. I also imagined this would be added in `constants.js`, but I cannot at this moment recall any other places where this would be used. In that case, let's go with @eugene-manuilov's suggestion. Thank you!

Reply:
Moved ✅